person.rb

```diff
@@ -3,6 +3,7 @@
 require_relative 'capitalize_decorator'
 
 class Person < Nameable
+  attr_reader :rentals
   attr_accessor :id, :name, :age
 
```

Comment:
- [ ] the :rentals should be attr_accessor since we will both read and write , kindly update it 🙏